e2e/paths/enterChildren_test.js

```diff
@@ -38,4 +45,52 @@ Scenario('completing entering child information in the c110a application', (I, e
   enterChildrenPage.defineAbilityToTakePartInProceedings();
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.applicationActions.enterChildren);
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  I.seeAnswerInTab(1, 'Child 1', 'Child\'s full name', 'Timothy');
```

Comment:
some of the spacing in this seems horrible. I don't think we should break half way though a sentence